src/lib/trx.js

```diff
@@ -305,6 +307,41 @@ export default class Trx {
         }).catch(err => callback(err));
     }
 
+    getAccountById(id = false, callback = false) {
+        if (!callback)
+            return this.injectPromise(this.getAccountById, id);
+
+        this.getAccountInfoById(id, {confirmed: true}, callback);
+    }
+
+    getAccountInfoById(id, options, callback) {
+        if (this.validator.notValid([
+            {
+                name: 'accountId',
+                type: 'hex',
+                value: id
+            },
+            {
+                name: 'accountId',
+                type: 'string',
+                lte: 32,
+                gte: 8,
+                value: id
+            }
+        ], callback))
+            return;
+
+        if (id.startsWith('0x')) {
+            id = id.slice(2);
+        }
+
+        this.tronWeb.fullNode.request(`wallet${options.confirmed ? 'solidity' : ''}/getaccountbyid`, {
```

Comment:
I think this will break if someone is using different server for full and solidity node. It should be something like:
```
this.tronWeb[options.confirmed ? 'solidityNode' : 'fullNode'].request(`wallet${options.confirmed ? 'solidity' : ''}/getaccountbyid`, {
```

Reply:
In the future, we could add the prefixes `wallet` and `walletsolidity` to the http providers and skip them in the requests.